runtime/testy/stateManager.py

```diff
@@ -152,9 +156,10 @@ def student_upload(self):
     self.processMapping[PROCESS_NAMES.TCP_PROCESS].send([ANSIBLE_COMMANDS.STUDENT_UPLOAD, True])
 
   def send_console(self, console_log):
+    return
```

Comment:
You probably know this already, but this `return` should probably be removed prior to merging.